test/Test.cpp

```diff
@@ -231,7 +231,7 @@ int main(int argc, char** argv)
 		java::lang::Integer integers4[] = { 1, 2, 3, 4 };
 		jni::Array<jint> test05(4, integers4);
 		for (int i = 0; i < test05.Length(); ++i)
-			printf("ArrayTest05[%ld],", (jint)test05[i]);
+			printf("ArrayTest05[%d],", (int)test05[i]);
```

Comment:
why is this needed?

Reply:
On Windows jint is typedefed as long, while on Mac as int. So the original code compiles fine on Windows, but gives warnings on Mac, since %ld means long, but is given int.